docs/apache-airflow-providers-amazon/connections/aws.rst

```diff
@@ -148,6 +148,18 @@ Snippet to create Connection and convert to URI
     os.environ[env_key] = conn_uri
     print(conn.test_connection())
 
+
+  .. warning:: When using airflow CLI there might be a need to add ``@`` when:
+
+    - login
+    - password,
+    - host
+    - port
+
+    are not given see example below. This is a known airflow limitation.
```

Comment:
Is there a comma or something missing here between `are not given` and  `see example below`?